Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -1,23 +1,20 @@
 # Widgets
 
 ## What is Widgets 
-Widgets Allows you to have easy access to a lot of different features, by selecting them from a menu.
-there is a total of nine-teen Widgets in the menu.
-This is designed in such a way that there is a ticket for all your needs.
-
-In the following the article there will be an explanation to each of these widgets.
-
+Widgets allows you to have easy access to a lot of different features, by selecting them from a menu. There is a total of nine-teen widgets in the menu. This is designed in such a way that there is a widget for all your needs.
 
+In the following article there will be an explanation to each of these widgets.
 
 ### The Hero Widget 
 
 ![movePage.jpg](images/The-Hero-Widget1.png)
 
 The Hero Widget allows you to create a hero element on your page, this includes features such as a heading, a rich text editor, a background image and the option to add buttons.
+
 You will also have the settings for your ***Hero Widget*** here, letting you have a nice overview of how you can customize the Widget to your liking.
```

Comment:
I'm not sure about this sentence - could you help me understand what you mean here? 🤔

Reply:
I just mean that you have the ability to set the Hero Widget up to your liking and that it is

Reply:
Alright! How about we do something like this instead then:

```suggestion
You will have a lot of settings for your ***Hero Widget***, letting you customize the widget to your liking.
```